resources/project.rb

```diff
@@ -9,13 +9,14 @@
 default_action :install
 
 attribute :project_dir, :kind_of => String, :name_attribute => true
-attribute :vendor, :kind_of => String, :name_attribute => true, :required => true
```

Comment:
Should be done now, vendor will still work. Let me make some changes to the readme file now :)